dist/gmusic.js

```diff
@@ -165,6 +165,15 @@ GMusic.Playback = {
   ALL_SHUFFLE: 'ALL_SHUFFLE',
```

Comment:
If we're going down the *woo hoo breaking change* path.

Then these constants can be removed.  We have new constants in the `SHUFFLE_MODES` and `REPEAT_MODES` namespaces